spec/visual/samples/sas

```diff
@@ -0,0 +1,71 @@
+* Example SAS script showing syntax highlighting;
+data population;
+/* 
+The first 10 lines of 2017 population data. Source: World Bank
+https://data.worldbank.org/indicator/sp.pop.totl
+Licence: CC BY-4.0

```

Comment:
Is this the correct format for the attribution?

Reply:
I've now added a link to the licence https://github.com/rouge-ruby/rouge/pull/1107/commits/3191ee40b262959258188a1c4661249f0daf3520 so I think this covers the bases as suggested in [CC's best practices document](https://wiki.creativecommons.org/wiki/best_practices_for_attribution).